src/cloudant/database.py

```diff
@@ -958,8 +955,8 @@ def security_url(self):
 
         :returns: Security document URL
         """
-        parts = ['_api', 'v2', 'db', self.database_name, '_security']
-        url = posixpath.join(self._database_host, *parts)
+        url = '/'.join((
+            self._database_host, '_api/v2/db', self.database_name, '_security'))
```

Comment:
This one example is different to all the others, why aren't `_api` `v2` and `db` separate parts here?